lib/ovirt_metrics/models/vm_device_history.rb

```diff
@@ -0,0 +1,10 @@
+module OvirtMetrics
+  class VmDeviceHistory < OvirtHistory
+    # have to rename the inheritance_column since this table has a column
+    # called "type"
+    self.inheritance_column = "other_type"
```

Comment:
How does this work for the other models like `OvirtMetrics::Calender` which also inherit from `OvirtHistory` ?
Those dont have the `inheritance_column`.
Do you only need this for the specs, because you create `VmDeviceHistory` ?
Maybe move this to the base class?